### DISCUSSION TITLE
Allow Gluetun-routed containers to speak to non-Gluetun-routed containers

### DISCUSSION THREAD

**Discussion by captainkev76** [qdm12/gluetun#1576] [Q&A]
Hey all!

I think this is a very commonly asked question, but I can't seem to find an answer that makes enough sense to let me sort it out. Hoping someone can help.

I have a docker-compose file which includes everything I want to be behind a VPN (Prowlarr and Qbittorrent). These all use the gluetun_default network. I have separate docker-compose files for other things like Sonarr, Radarr and FlareSolverr. They all share the "docker_default" network. The Prowlarr container can't speak to the FlareSolverr container, presumably as they are on different networks.

Is there a simple way to make this work without having to put everything behind gluetun?

Thanks in advance!

**Comment by gmillerd** [comment]
So you just have two docker-composes without using the gluetun networking? You might want to share your yml(s), in part, like other issues are reported.

- https://github.com/qdm12/gluetun/wiki/Connect-a-container-to-gluetun
- https://github.com/qdm12/gluetun/wiki/Inter-containers-networking
